pkg/uefivars/boot/efibootmgr.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2015-2020 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+package boot
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"unicode/utf16"
+)
+
+// efibootmgr's output is thoroughly horrible
+// read efi variables directly instead of using efibootmgr to show entries
+
+// Add a boot entry, shelling out to efibootmgr. Entry is set as primary.
+func AddBootEntry(b BootEntry) {
```

Comment:
could we leave the os/exec command wrappers somewhere else or in a different package?

Reply:
Yes, we can remove the wrappers, put them elsewhere, use a build tag (actually thought I had added one), or whatever. I didn't implement write support natively, and I'm not sure if it's as simple as writing to the files under /sys or not.

Reply:
yeah it probably isn't. separate package is probably preferable, build tags are somewhat hard to deal with in some bazel environments.

Reply:
moved